src/vs/workbench/services/telemetry/browser/telemetryService.ts

```diff
@@ -40,15 +40,9 @@ export class TelemetryService extends Disposable implements ITelemetryService {
 
 		if (supportsTelemetry(productService, environmentService) && productService.aiConfig?.asimovKey && productService.aiConfig?.ariaKey) {
 			// If remote server is present send telemetry through that, else use the client side appender
-			const internalTesting = configurationService.getValue<boolean>('telemetry.internalTesting');
 			const appenders = [];
-			if (internalTesting || productService.aiConfig?.preferAria) {
-				const telemetryProvider: ITelemetryAppender = remoteAgentService.getConnection() !== null ? { log: remoteAgentService.logTelemetry.bind(remoteAgentService), flush: remoteAgentService.flushTelemetry.bind(remoteAgentService) } : new OneDataSystemWebAppender('monacoworkbench', null, productService.aiConfig?.ariaKey);
-				appenders.push(telemetryProvider);
-			} else {
-				const telemetryProvider: ITelemetryAppender = remoteAgentService.getConnection() !== null ? { log: remoteAgentService.logTelemetry.bind(remoteAgentService), flush: remoteAgentService.flushTelemetry.bind(remoteAgentService) } : new WebAppInsightsAppender('monacoworkbench', productService.aiConfig?.asimovKey);
-				appenders.push(telemetryProvider);
-			}
+			const telemetryProvider: ITelemetryAppender = new GitpodInsightsAppender();
+			appenders.push(telemetryProvider);
```

Comment:
I think we need this condition ` remoteAgentService.getConnection() !== null ? ...` , if not all telemetry events will be send using the client :thinking:

Reply:
Hey, JP, I'm thinking about why we need this `if condition` since, we are only browser here